src/index.ts

```diff
@@ -52,10 +52,18 @@ export class APIClient {
 	}
 
 	public async authenticate(data: AuthenticateData) {
-		const response: AxiosResponse<APIAuthenticateResponse> = await axios.post(`${this.apiBase}/authenticate`, data, this.baseConfig);
-		this.token = response.data.token;
+		const response: AxiosResponse<{ status: APIAuthenticateResponse }> = await axios.post(`${this.apiBase}/authenticate`, data, this.baseConfig);
+		this.token = response.data.status.token;
```

Comment:
This is actually just `response.data.token`:

![image](https://user-images.githubusercontent.com/10330923/92414325-a0276900-f14b-11ea-949c-9fdc51a2c1dc.png)

`response.data` is the wrapping object of the above, so can you undo these changes?